lambda_function.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+def my_handler(event, context):
+     s3=boto3.client("s3")
+     if event:
+        print("Event : ",event)
+        fil_obj=event["Record"][0]
+        filename=str(fil_obj['s3']['object']['key'])
+        print("ilename:",filename)
+        fileobj=s3.get_object(Bucket=Sourcebucket,Key=filename)

```

Comment:
Is this working? I can see that lambda iam role doesn't have any permission related to s3